### ISSUE TITLE
Auckland page

### ISSUE THREAD

**Issue by cameronmclean** [resbaz/resbaz-2016-02-01#17]
Hi All!
Can i get an Auckland stub - I should have time this week to start filling in the local info and links.
Cheers!
Cam

**Comment by katieannemills** [comment]
No problem - [here's your stub](https://feb2016.resbaz.com/auckland/); you can PR `_posts/2016-02-01-auckland.html` to fill in the text, or add update requests to this thread and I'll add them for you. Also, let me know a few locations:
- the exact coordinates you want the Auckland pin to appear at on the global map
- a few local points of interest for attendees: lesson rooms, social events, recommended hotels, whatever you want to provide a map of for your attendees. Please include lat/long, and a tweet-length description of what each place is.

Let me know how else I can help, and welcome aboard!

**Comment by cameronmclean** [comment]
Awesome - thanks Bill.
I'll post co-ords and submit a PR in the next few days.
Also - in case it's helpful for others - I couldn't serve a local copy of the site using Jekyll 3.0 on MacOS 10.11.1 (El Capitan) - trying to view posts via the permalink (e.g `localhost/auckland/` etc) always gave 404s. Switching back to Jekyll 2.5.3 on my machine had no problems however...

**Comment by sungeunbae** [comment]
BTW, New Zealand ResBaz are 2016-02-02, not 2016-02-01. To avoid any confusion, it'd be best to update the url. Thanks

**Comment by katieannemills** [comment]
thanks, @sungeunbae! I think @tjdett is driving on the domain, hopefully he can help.

**Comment by DamienIrving** [comment]
@tjdett What do you think the best solution is? Formats like 2016-02-01 are great if you've got lots and lots of files (because they sort in order), but since there will be at most two ResBaz events per year, it's not that many to sort. Perhaps https://feb2016.resbaz.com/ would do the trick?

In most of our advertising we've simply used resbaz.com, but just in case we'd need to have https://2016-02-01.resbaz.com/ (and https://2016-02-01.resbaz.com/melbourne) redirect to https://feb2016.resbaz.com/ (and https://feb2016.resbaz.com/melbourne). The other sites haven't been using their pages yet, so Melbourne is the only critical redirect.

**Comment by DamienIrving** [comment]
Of course, this format would be an issue if we ever ran an event that straddled two months (e.g. Jan 31 - Feb 2), but I think we can cross that bridge if/when we come to it (e.g. jan-feb2017 would probably be fine).

**Comment by cameronmclean** [comment]
Auckland lat/longs - For now we only need the Owen G Glenn bulding @ Auckland Uni. This can be used for both the global pin, and the local page.
Goolge gives the co-ords as the following

```
Owen G Glenn Bldg
Auckland, 1010
-36.853083, 174.771265
```

I just made a PR for the current Auckland event info.
As for the domain name - for now I have placed a Note on the Auckland page highlighting the local event dates - I think that should cover us for now...

**Comment by katieannemills** [comment]
@cameronmclean awesome, thanks! I just updated your pins with those coords.

**Comment by mikblack** [comment]
@cameronmclean  - I "borrowed" your Auckland edits for the Dunedin page (https://feb2016.resbaz.com/dunedin/).  Many thanks - I owe you a beverage.  :)
